default.nix

```diff
@@ -1,5 +1,12 @@
-{ pkgs ? import <nixpkgs> { }
-, ghc ? "ghc8102"
+{ nixpkgsSrc ?
+    builtins.fetchTarball {
+      # The test-fix-shellFor-doCheck-minimal branch of cdepillabout/nixpkgs.
+      # This is built on a recent version of nixpkgs master as of 2020-11-20.
+      # This has a single commit that adds some functionality to shellFor.
+      url = "https://github.com/cdepillabout/nixpkgs/archive/47d19bef864.tar.gz";
```

Comment:
This branch just adds the following commit: https://github.com/cdepillabout/nixpkgs/commit/47d19bef86481df4f52c2bf909c23940cf98973a